### PR TITLE
Fixed missing member variable doc comment

### PR DESCRIPTION
**Pull request by MPParsley**

### CODE REVIEW

File: tests/src/Kernel/Entity/OgStandardReferenceItemTest.php [hunk 1]
```diff
@@ -20,8 +20,25 @@ class OgStandardReferenceItemTest extends KernelTestBase {
...
+   * The machine name for a field.
+   *
+   * @var array
+   */
   protected $fieldName;
+
+  /**
+   * A list of groups.
+   *
+   * @var EntityTest[]
```
Comment (MPParsley) [inline tests/src/Kernel/Entity/OgStandardReferenceItemTest.php]:
```suggestion
   * @var \Drupal\entity_test\Entity\EntityTest[]
```

File: tests/src/Kernel/Entity/OgStandardReferenceItemTest.php [hunk 2]
```diff
@@ -20,8 +20,25 @@ class OgStandardReferenceItemTest extends KernelTestBase {
...
+   * A list of bundles.
+   *
+   * @var string[]
+   */
   protected $bundles;
+
+  /**
+   * The machine name for a field.
+   *
+   * @var array
```
Comment (MPParsley) [inline tests/src/Kernel/Entity/OgStandardReferenceItemTest.php]:
```suggestion
   * @var string[]
```

File: tests/src/Kernel/Entity/OgStandardReferenceItemTest.php [hunk 3]
```diff
@@ -20,8 +20,25 @@ class OgStandardReferenceItemTest extends KernelTestBase {
...
+   * A list of bundles.
+   *
+   * @var string[]
+   */
   protected $bundles;
+
+  /**
+   * The machine name for a field.
+   *
+   * @var string[]
```
Comment (amitaibu) [inline tests/src/Kernel/Entity/OgStandardReferenceItemTest.php]:
```suggestion
   * @var string
```

Comment (MPParsley) [reply tests/src/Kernel/Entity/OgStandardReferenceItemTest.php]: Whoops, copy/paste :-)

**Comment by amitaibu** [comment]
:+1: MWG (Merge When Green :smile_cat: )

**Pull request merged**